### DISCUSSION TITLE
Adding plugins to neuron.dhall breaks automatic publishing

### DISCUSSION THREAD

**Discussion by brandonlucasgreen** [srid/neuron#604] [Ideas]
Hi! I am fairly new to neuron. I set an instance up in Github with automatic publishing to my website. I’d like to use the directory tree `dirtree` plugin, but when I try to add support for it in my neuron.dhall file, the publish.yaml script seems to consistently fail. However, I am sure my neuron.dhall file is formatted correctly based on the neuron documentation. Is this a known issue or something that can be worked around / resolved?

Here’s the error that occurs:
```
Run ./neuron -d $PWD gen
5 !! Config file error
6 Error: Invalid input
7  
8  neuron.dhall:5:3:
9  |
10 5 | , plugins = [ “neuronignore”, “links”, “uptree”, “feed”, “dirtree”, “tags” ]
11 |   ^
12 unexpected 'p'
13 expecting whitespace or }
14 touch: cannot touch '.neuron/output/.nojekyll': No such file or directory
15 Error: Process completed with exit code 1.
```

**Comment by srid** [comment]
Those quotation marks look strange. Can you try with normal quotes?

Compare

> “

vs 

> "

**Comment by brandonlucasgreen** [comment]
Argh, that was it. I can't believe I missed that. Thanks!
